plugins/cilium-cni/chaining/awscni/network.go

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package awscni
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+
+	"github.com/cilium/cilium/pkg/datapath/linux/linux_defaults"
+	"github.com/cilium/cilium/pkg/datapath/linux/route"
+)
+
+// awsCNIVLANIfacePrefix is the prefix used by the AWS CNI for building
+// interface names for SGP Pod VLAN interfaces
+const awsCNIVLANIfacePrefix = "vlan.eth."
+
+// buildSGPPodVLANIfaceName returns the name for the Pod associated with
+// the VLAN ID vlanID
+func buildSGPPodVLANIfaceName(vlanID string) string {
+	return awsCNIVLANIfacePrefix + vlanID
+}
+
+// disableIfaceRPFilter disables the `rp_filter` network setting for the
+// network interface iface
+func disableIfaceRPFilter(iface string) error {
+	return os.WriteFile(
```

Comment:
We need to disable reverse path filtering by the kernel and handle it instead with iptables.